indexer/feature_utils.cpp

```diff
@@ -472,4 +482,62 @@ bool HasToilets(TypesHolder const & types)
   return isToiletsType(types);
 }
 
+string FormatStars(uint8_t starsCount)
+{
+  std::string stars;
+  for (int i = 0; i < starsCount && i < kMaxStarsCount; ++i)
+    stars.append(kStarSymbol);
+  return stars;
+}
+
+string FormatElevation(string_view elevation)
+{
+  if (!elevation.empty())
+  {
+    double value;
+    if (strings::to_double(elevation, value))
+        return std::string{kMountainSymbol} + platform::Distance::FormatAltitude(value);
+    else
+      LOG(LWARNING, ("Invalid elevation metadata:", elevation));
+  }
+  return {};
+}
+
+string_view constexpr kWlan = "wlan";
+string_view constexpr kWired = "wired";
+string_view constexpr kTerminal = "terminal";
+string_view constexpr kYes = "yes";
+string_view constexpr kNo = "no";
+
+string_view DebugPrint(Internet internet)
```

Comment:
DebugPrint should always return std::string, that causes CI failures. Please also check other comments.